treeview/js/main.js

```diff
@@ -35,8 +37,8 @@
             var showAnnotation, id_parameter, filterUrl, filterValue, columnName, parentAppExists, selected_option;
             var annotated_term  = "";
             var annotated_terms = [],
-                urlParams       = {}, // key/values from uri
-                urlParamNames   = [], // keys defined in url
+                // urlParams       = {}, // key/values from uri
+                // urlParamNames   = [], // keys defined in url
```

Comment:
please uncomment after removing the definition at the top of the file.

Reply:
Made required changes